lib/titiler-pgstac-api/index.ts

```diff
@@ -0,0 +1,113 @@
+import {
+    Stack,
+    aws_iam as iam,
+    aws_ec2 as ec2,
+    aws_rds as rds,
+    aws_lambda as lambda,
+    aws_secretsmanager as secretsmanager,
+    CfnOutput,
+    Duration,
+    aws_logs,
+  } from "aws-cdk-lib";
+  import { HttpApi } from "@aws-cdk/aws-apigatewayv2-alpha";
+  import { HttpLambdaIntegration } from "@aws-cdk/aws-apigatewayv2-integrations-alpha";
+  import { Construct } from "constructs";
+  
+  export class TitilerPgstacApiLambda extends Construct {
+    readonly url: string;
+  
+    constructor(scope: Construct, id: string, props: TitilerPgStacApiLambdaProps) {
+      super(scope, id);
+  
+      const titilerPgstacEnv = {
+          "CPL_VSIL_CURL_ALLOWED_EXTENSIONS": ".tif,.TIF,.tiff",
+          "GDAL_CACHEMAX": "200", 
+          "GDAL_DISABLE_READDIR_ON_OPEN": "EMPTY_DIR",
+          "GDAL_INGESTED_BYTES_AT_OPEN": "32768",
+          "GDAL_HTTP_MERGE_CONSECUTIVE_RANGES": "YES",
+          "GDAL_HTTP_MULTIPLEX": "YES",
+          "GDAL_HTTP_VERSION": "2",
+          "PYTHONWARNINGS": "ignore",
+          "VSI_CACHE": "TRUE",
+          "VSI_CACHE_SIZE": "5000000", 
+          "DB_MIN_CONN_SIZE": "1",
+          "DB_MAX_CONN_SIZE": "1",
+          "PGSTAC_SECRET_ARN": props.dbSecret.secretArn,
+      }
+    
+      
+      const handler = new lambda.Function(this, "lambda", {
+        handler: "handler.handler",
+        runtime: lambda.Runtime.PYTHON_3_8,
+        code: lambda.Code.fromDockerBuild(__dirname, {
+          file: "runtime/Dockerfile",
```

Comment:
To ensure that I understand correctly, if we create assets bundle but don't manually go through at trim it down with the `find ... | xargs rm` commands within the `Dockerfile`, the assets zip will be too large?